scripts/ci/local_ci_extract_tests.sh

```diff
@@ -21,19 +21,25 @@ set -euxo pipefail
 
 MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" || exit 1; pwd )"
 
-AIRFLOW_SOURCES=$(cd "${MY_DIR}/../../.." || exit 1; pwd)
-export AIRFLOW_SOURCES
+# shellcheck source=scripts/ci/_utils.sh
+. "${MY_DIR}/_utils.sh"
 
-gosu "${AIRFLOW_USER}" nosetests --collect-only --with-xunit --xunit-file="${HOME}/all_tests.xml"
+basic_sanity_checks
 
-gosu "${AIRFLOW_USER}" \
-    python "${AIRFLOW_SOURCES}/tests/test_utils/get_all_tests.py" \
-                    "${HOME}/all_tests.xml" >"${HOME}/all_tests.txt"; \
+script_start
 
-echo ". ${HOME}/.bash_completion" >> "${HOME}/.bashrc"
+export PYTHON_VERSION=3.5
```

Comment:
Can we add a comment here on why we have hard-coded Py 3.5 please?

Reply:
Yep. It's part of the earlier commit #6536  - i tried to make sure it is noticed in the comment (please review only last commit) but it's easy to misse.

Reply:
It's removed now altogether as we found a better way of extracting tests :)/